src/site/pages/digital/src/containers/DigitalHeader/index.tsx

```diff
@@ -0,0 +1,30 @@
+import {Header} from "shared/containers/Header";
+import {HeaderPopups} from "shared/state/Header/state";
+import {Dropdown} from "shared/containers/Header/Right/Dropdown";
+
+import {CircuitInfoHelpers} from "shared/utils/CircuitInfoHelpers";
+
+type Utility = {
+    popupName: HeaderPopups;
+    img: string;
+    text: string;
+}
+
+function getExtraUtilities(): Utility[] {
+    return [
+        { 
+            popupName: "expr_to_circuit",
+            img: "img/icons/bool_expr_input_icon.svg",
+            text: "Boolean Expression to Circuit"
+        }
+    ];
+}
```

Comment:
You can just define 
```ts
const ExtraUtilties = [
        { 
            popupName: "expr_to_circuit",
            img: "img/icons/bool_expr_input_icon.svg",
            text: "Boolean Expression to Circuit"
        }
    ];
```

and then the type is implicit

Reply:
Changed to the const declaration rather than the function returning, but I still had to specify the Utility[] since VS Code didn't seem to realize that it was a Utility type.